app/components/createContainer/createContainerController.js

```diff
@@ -232,7 +233,174 @@ function ($q, $scope, $state, $stateParams, $filter, Container, ContainerHelper,
+          volume.type = "bind";
+        }
+        if (arr[2] && arr[2] === 'ro') {
+          volume.readOnly = true;
+        }*/
+        $scope.formValues.Volumes.push(volume);
+      }
+      // Add network
+      $scope.config.NetworkingConfig = {
+        EndpointsConfig: {}
+      };
+      if ($scope.config.HostConfig.NetworkMode.indexOf('container:') === 0) {
+        $scope.config.HostConfig.NetworkMode = 'container';
+        $scope.formValues.NetworkContainer = $scope.config.HostConfig.NetworkMode.split(/^container:/)[1];
+      }
+      if (d.NetworkSettings.Networks[$scope.config.HostConfig.NetworkMode]) {
+        console.log(d.NetworkSettings.Networks[$scope.config.HostConfig.NetworkMode]);
+        if (d.NetworkSettings.Networks[$scope.config.HostConfig.NetworkMode].IPAMConfig.IPv4Address) {
+          $scope.formValues.IPv4 = d.NetworkSettings.Networks[$scope.config.HostConfig.NetworkMode].IPAMConfig.IPv4Address;
+        }
+        if (d.NetworkSettings.Networks[$scope.config.HostConfig.NetworkMode].IPAMConfig.IPv6Address) {
+          $scope.formValues.IPv6 = d.NetworkSettings.Networks[$scope.config.HostConfig.NetworkMode].IPAMConfig.IPv6Address;
+        }
+      }
+      $scope.config.NetworkingConfig.EndpointsConfig = d.NetworkSettings.Networks;
+
+      // Add Env
+      var envArr = [];
+      for (var e in $scope.config.Env) {
+        var arr = $scope.config.Env[e].split(/=(.+)/);
+        envArr.push({"name": arr[0], "value": arr[1]});
+      }
+      $scope.config.Env = envArr;
+
+      // Add ExtraHost
+      for (var h in $scope.config.HostConfig.ExtraHosts) {
+        $scope.formValues.ExtraHosts.push({"value": $scope.config.HostConfig.ExtraHosts[h]});
+        $scope.config.HostConfig.ExtraHosts = [];
+      }
+
+      // Add labels
+      for (var l in $scope.config.Labels) {
+        $scope.formValues.Labels.push({ name: l, value: $scope.config.Labels[l]});
+      }
+
+      // Add Console
+      if ($scope.config.OpenStdin && $scope.config.Tty) {
+        $scope.formValues.Console = 'both';
+      } else if (!$scope.config.OpenStdin && $scope.config.Tty) {
+        $scope.formValues.Console = 'tty';
+      } else if ($scope.config.OpenStdin && !$scope.config.Tty) {
+        $scope.formValues.Console = 'interactive';
+      } else if (!$scope.config.OpenStdin && !$scope.config.Tty) {
+        $scope.formValues.Console = 'none';
+      }
+
+      // Add Devices
+      var path = [];
+      for (var dev in $scope.config.HostConfig.Devices) {
+        var device = $scope.config.HostConfig.Devices[dev];
+        path.push({"pathOnHost": device.PathOnHost, "pathInContainer": device.PathInContainer});
+      }
+      $scope.config.HostConfig.Devices = path;
+
+      // Add Ownership
+      /*if (d.Portainer && d.Portainer.ResourceControl) {
+        ControllerDataPipeline.setAccessControlFormData(....);
+      }*/
+  };
+
+      // Add name
+      $scope.config.name = d.Name.replace(/^\//g, '');
```

Comment:
With compose, got same issue
Once the IPv4 javascript error resolved, it's OK, so I think just javascript stopped and not fill the form